pkgs/development/python-modules/pyzstd/default.nix

```diff
@@ -25,8 +28,17 @@ buildPythonPackage rec {
         --replace-fail '"setuptools>=64,<74"' '"setuptools"'
   '';
 
+  dependencies = [
```

Comment:
Should be below `build-system`.

Reply:
I'd personally also keep `dependencies` and `buildInputs` grouped together, as you suggest.

But this feels like a case of https://github.com/NixOS/nixpkgs/issues/387072, where the feedback comes across as overly nit-picky, especially when no rational is provided :wink:

Reply:
IMHO it doesn't *really* matter, but I'll listen to the feedback nonetheless. I'm fairly new to Python packages in Nixpkgs, let alone contributing to it, so please forgive me if I don't know about unwritten coding conventions.